docs/source/command_line.rst

```diff
@@ -480,6 +480,11 @@ of the above sections.
            # 'items' now has type List[List[str]]
            ...
 
+.. option:: --disable-error-code CODE
```

Comment:
The command line option is already documented on line 556.

Reply:
OK removed the duplicate entry I added